jetty/src/main/scala/org/http4s/server/jetty/JettyBuilder.scala

```diff
@@ -66,15 +65,14 @@ sealed class JettyBuilder private(
   override def bindSocketAddress(socketAddress: InetSocketAddress): JettyBuilder =
     copy(socketAddress = socketAddress)
 
-  override def withServiceExecutor(serviceExecutor: ExecutorService): JettyBuilder =
-    copy(serviceExecutor = serviceExecutor)
+  override def withExecutionContext(executionContext: ExecutionContext): Self =
```

Comment:
Actually, one more thing: we could preserve `withServiceExecutor` as a deprecated method, and construct an EC from it and delegate to `withExecutionContext`.  That will help people transition.